auditbeat/module/auditd/_meta/kibana/6/dashboard/auditbeat-kernel-overview.json

```diff
@@ -54,7 +54,7 @@
                                     }
                                 ], 
                                 "point_size": 1, 
-                                "seperate_axis": 0, 
+                                "separate_axis": 0, 
```

Comment:
we should probably not change things in the dashboards as these are based on an API. So the API might have a typo and we break it with these changes. This applies to everything under the `_kibana` directory.